system-variables.md

```diff
@@ -1642,22 +1644,22 @@ This variable is an alias for `transaction_isolation`.
 - Default value: (string)
 - This variable returns additional details about the TiDB version. For example, 'TiDB Server (Apache License 2.0) Community Edition, MySQL 5.7 compatible'.
 
-### version_compile_os
+### version_compile_machine
 
 - Scope: NONE
 - Default value: (string)
-- This variable returns the name of the OS on which TiDB is running.
+- This variable returns the name of the CPU architecture on which TiDB is running.
 
-### version_compile_machine
+### version_compile_os
 
 - Scope: NONE
 - Default value: (string)
-- This variable returns the name of the CPU architecture on which TiDB is running.
+- This variable returns the name of the OS on which TiDB is running.
 
 ### wait_timeout
 
 - Scope: SESSION | GLOBAL
-- Default value: `0`
+- Default value: `28800`
```

Comment:
We always treat the default value change with extra care. From which version this default value is changed? If it is changed from v5.3.0, shall we let the QA team @seiya-annie know and add it to v5.3.0 release notes?

Reply:
I took a look at the PR, and it's not cherry picked: https://github.com/pingcap/tidb/pull/29569
So it is master only right now.

It might be worth docs having a script to detect defaults changes. I am not specifically changing the default here - it has already been changed.

Reply:
Translated into Chinese in https://github.com/pingcap/docs-cn/pull/8076